section1/reflection.md

```diff
@@ -1,19 +1,42 @@
+like menUsedToLookBetterInHats = true  
+
+3. How would you log the string `"Hello World!"` to the console?  
+I think this is a trick question? Because you use quotes to define something as a string using them within the string can confuse things so you have to be careful and use \ to mark special characters. So in order to log this exact string you'd use
+console.log('\`"Hello World!"\`').  
+If I'm overthinking this and the back ticks are just here to `format soemthing as code` then you could simply use single quotes to enclose the double quotes as in  
+console.log('"Hello World!"').
+4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?  
+// to mark off the rest of a line as a comment (everything after the //)  
+/* to mark off a section of a line */  
+Comments are useful for collaboration since they give collaborators a sense of what a section of code is supposed to do or what's going wrong with it.
+They are also useful for communicating with your future self who has entirely forgotton why you put in this weird line of code.  
+
+5. In your own words, what is a variable? How would you explain it to a 5 year old?  
+A variable is a piece of information for the computer that might change. You mark it as a variable to say "this is 5 now but maybe later it'll be 7 or 3 or even a word like 'apple'"  
+
+6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?  
+I'm using a pomodoro app right now that probably keeps track of variables like (variable names changed to protect the innocent):  
+pomsCompleted which would be a number variable of my completed pomodoros that day.  
+pomTask which would be a string that let you specify what this pom was for broadly speaking  
+pomBreak another number variable keeping seperate track of what break between poms this is for the day since some pom breaks are longer than others.  
+You could theoretically do everything you'd do with PomBreak by performing mathematical functions on pomsCompleted but I probably wouldn't for various reasons.  
+7. In your own words, explain what concatenation is.  
+Shoving different strings and variables into the same spot to put them in one place for the user to read them together.  
+
+8. Think of a site or app you use frequently. Where do you think the developers used concatenation?  
+Every time I log into Twitter lately (a thing I try very hard not to do) it asks if my e-mail address is the same in a format that I bet looks like
+"is ${userEmail} still your email address?"  
+
+9. What questions do you still have about the work we've done so far? (not having a question is not an option)  
+In the readings on Git we were doing for our session 2 homework it ended with the guy saying you should always use git from the command line and
```

Comment:
Great question. While at Turing, strongly discouraged to use any GUI (if you use it for your own learning purposes, cool but what I mean is in practice for actual work)- we want you to understand the foundations of how it works and be fluent using the Terminal. Most not all devs and companies will look for your proficiency in this and expect this on the job. And since you'll be working in such collaborative environments, we don't want the fact that one person uses something "special" to slow down, confuse, etc other teammates.